slu/slu/utils/preprocessing.py

```diff
@@ -74,9 +74,11 @@ def make_data_column_uniform(data_frame: pd.DataFrame) -> pd.DataFrame:
     ):
         if isinstance(row[const.ALTERNATIVES], str):
             data = json.loads(row[const.ALTERNATIVES])
+            if isinstance(data, str):
```

Comment:
Why would `json.loads` yield a string? Shouldn't it strictly give a `dict`?

Reply:
Production calls in vodafone contain noisy transcripts, where ASR alternatives seem to be dumped twice. The second `if` condition is supposed to fix this by performing another json.loads(data). 

Case example:
<img width="745" alt="image" src="https://user-images.githubusercontent.com/48950252/222171914-67ebb7fe-fb08-49c0-82b8-c28f6285331e.png">